### PR TITLE
Add German HOWTO file

### PR DESCRIPTION
**Pull request by Squirkyy**
## What does this PR do?
Add ONE HOWTO file (german) + ONE CODEOFCONDUCT file (german) and changes EIGHT HOWTO files
### Description
I added a German Translation for the HOWTO file and added the option to switch to German in all the other files.
I also added the German Translation for the Code of Conduct.
### Why is this valuable (or not)?
It helps the German learning community.

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [X] Search for duplicates.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by eshellman** [comment]
please resolve conflicts.

File: HOWTO-de.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Lese das hier auch in anderen Sprachen: [English](HOWTO.md), [Français](HOWTO-fr.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md) [Español](HOWTO-es.md), [فارسی](HOWTO-fa_IR.md)
```
Comment (davorpa) [inline HOWTO-de.md:1]: Please, mantain the same order of items like you can see in other HOWTOs. It contributes to have a good UX.

```markdown
[Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md).
```

File: HOWTO-de.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+* [Github Hallo Welt](https://guides.github.com/activities/hello-world/) *(auf Englisch)*
+* [Youtube - Tutorial Github für Anfänger](https://www.youtube.com/watch?v=0fKg7e37bQE) *(auf Englisch)*
+* [Youtube - So forkst du ein Github-Repo und sendest einen Pull-Request](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(auf Englisch)*
+* [Youtube - Github Markdown](https://www.youtube.com/watch?v=HUBNt18RFbo) *(auf Englisch)*
+
+
+Habe keine Angst eine Frage zu stellen; jeder muss irgendwo anfangen und den ersten Pull Request machen. Du könntest der 1000. sein.
+
+Selbst wenn du ein erfahrener Open-Source-Mitwirkender sind, gibt es Dinge, die dich ins Straucheln bringen könnten. Sobald du deinen PR eingereicht hast, führt GitHub Actions einen Linter aus und findet oft kleine Probleme von Absätzen oder Alphabetisierung. Wenn du eine grüne Schaltfläche erhältst, ist alles zur Überprüfung bereit, aber wenn das nicht so ist, klick unter dem fehlgeschlagenen Häkchen auf "Details", um herauszufinden, was dem Linter nicht gefallen hat. Behebe das Problem und füge deinen PR einen Commit hinzu.
```
Comment (davorpa) [inline HOWTO-de.md:15]: It's safisfy #5644

**Review by davorpa — CHANGES_REQUESTED** [review]
Some feedback and changes:

- Conficts are due to recent added russian HOWTO #5669 . The new text was added at the end on paragraph like you are also doing.

File: HOWTO-de.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Wenn du dir nicht sicher bist, ob die Ressource, die du hinzufügen möchtest, für Free-Programming-Books geeignet ist, lese die Richtlinien in [CONTRIBUTING](CONTRIBUTING.md).
```
Comment (davorpa) [inline HOWTO-de.md:17]: To be coordinated with @lupinitylabs in their translation (see #5720) ... it's safe change crosslink to `[CONTRIBUTING](CONTRIBUTING-de.md)` too :wink: 

```markdown
Wenn du dir nicht sicher bist, ob die Ressource, die du hinzufügen möchtest, für Free-Programming-Books geeignet ist, lese die Richtlinien in [CONTRIBUTING](CONTRIBUTING-de.md).
```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT-de.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+diese Prinzipien gerecht und einheitlich auf jeden Aspekt des Projektmanagements anzuwenden.
+Projektverantwortliche, die sich nicht nach dem Verhaltenskodex richten oder ihn nicht durchsetzen,
+können dauerhaft aus dem Projektteam ausgeschlossen werden.
+
+Dieser Verhaltenskodex gilt sowohl innerhalb des Projektbereichs als auch in
+öffentlichen Bereichen, wenn eine Person das Projekt oder seine Gemeinschaft repräsentiert.
+
+
+Fälle von missbräuchlichem, belästigendem oder anderweitig nicht akzeptablen Verhalten
+können den Projektverantwortlichen unter [victorfelder@gmail.com] gemeldet werden.
```
[the remark this answers is not in the document]
Comment (Squirkyy) [reply CODE_OF_CONDUCT-de.md]: Changed it!

File: HOWTO-de.md (hunk 3 above)
Comment (Squirkyy) [reply HOWTO-de.md]: Edited it!

File: HOWTO-de.md (hunk 1 above)
Comment (Squirkyy) [reply HOWTO-de.md]: Changed it! :)

**Comment by Squirkyy** [comment]
I changed everything you have mentioned. Anything else that needs changing?

**Review by davorpa — CHANGES_REQUESTED** [review]
> I changed everything you have mentioned. Anything else that needs changing?

It seems that all other HOWTO files was lost :worried: on resync upstream. SHAs:
- https://github.com/EbookFoundation/free-programming-books/pull/5684/commits/7cd89677db2cd888c774032ebfa695079916ed3e
- https://github.com/EbookFoundation/free-programming-books/pull/5684/commits/f2d6969a6a67d2178c95e5f8d851389642e8ae4f

:question: Do you know how to deal with? Maybe do some cherry picks from your firsts pushed commits 🤔

**Comment by eshellman** [comment]
looks like you added inadvertent punctuation - look at diffs

**Comment by Squirkyy** [comment]
I think I'll just fork the repo again and commit another PR. Its probably easier and faster

**Comment by davorpa** [comment]
Work moved to #5819

**Pull request closed without merging**